### PR TITLE
Add Participant Box

### PR DESCRIPTION
**Pull request by ahwnsals1234**
First draft of the Add Participant box. Has a text field for the name of the participants to add and a field to type the Gordon email of the participant.

### CODE REVIEW

**Comment by ahwnsals1234** [comment]
Not yet ready for a review, need a couple more updates to be made before being eligible for a code review.

File: src/views/HousingLottery/index.jsx [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
 
 const HousingLottery = () => {
-  return <PreferredHall />;
+
+
+
+  return (
+    <Grid container spacing={2} justifyContent="center">
+      <Grid item xs={12} lg={10}>
+        <PreferredHallsCard/>
```
Comment (ArabellaJi) [inline src/views/HousingLottery/index.jsx:15]: Is there a reason you remove `<PreferredHall />` and put a `<PreferredHallsCard/>` here? I didn't see you define `PreferredHallsCard` anywhere, but I'm surprised that in vm it still loads the Preferred Hall section.

File: src/views/HousingLottery/studentView/StudentApplicants/ApplicantFields/index.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      <TextField
+        type="email"
+        variant='outlined'
+        color='secondary'
+        label="Email"
+        onChange={(e) => handleFieldChange('email', e.target.value)}
+        value={applicant.email}
+        required
+        error={!applicant.email.endsWith('@gordon.edu')}
+        helperText={!applicant.email.endsWith('@gordon.edu') ? 'Not a Valid Gordon Email' : '*Required'}
```
Comment (ArabellaJi) [inline src/views/HousingLottery/studentView/StudentApplicants/ApplicantFields/index.jsx:44]: Can you add a check for empty string, so when there is no input, the text field would not be red like now?
image

File: src/views/HousingLottery/studentView/StudentApplicants/index.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  };
+
+  //   We will be having set number of applicants for the MVP and for now.
+  // const addApplicant = () => {
+  //   setApplicants([...applicants, { firstName: '', lastName: '', email: '' }]);
+  //   setEmails([...emails, '']);
+  // };
+
+  const handleSubmit = () => {
+    console.log(emails);
```
Comment (ArabellaJi) [inline src/views/HousingLottery/studentView/StudentApplicants/index.jsx:34]: You can keep this console log, but please remove all other console logs and commented-out code to make you files clean.

**Review by ArabellaJi — CHANGES_REQUESTED** [review]
Thank you for working on this! Just a few simple things and you should be good to go.

**Comment by andrew-wzj** [comment]
Once you clean the code and conflicts mentioned in the `HousingLottery.module.scss` and `index.jsx`, your branch would be ready to merge. Thank you!

File: src/views/HousingLottery/index.jsx (hunk 1 above)
Comment (ahwnsals1234) [reply src/views/HousingLottery/index.jsx:15]: I think I accidentally changed that after trying couple of things. I'll change it back to <PreferredHall />!!

File: src/views/HousingLottery/studentView/StudentApplicants/index.jsx (hunk 3 above)
Comment (ahwnsals1234) [reply src/views/HousingLottery/studentView/StudentApplicants/index.jsx:34]: Will do!!

**Pull request merged**